OpenKh.Tools.Kh2MdlxEditor/Views/Model_Control.xaml

```diff
@@ -84,9 +85,20 @@
                 </ListView.View>
             </ListView>
             
-            <GridSplitter Grid.Row="1" VerticalAlignment="Stretch" />
+            <DockPanel Grid.Row="1">
+                <!-- Buttons to reorder the meshes -->
+                <Button DockPanel.Dock="Right" Margin="5" HorizontalAlignment="Right" Width="40" Height="25" Click="Mesh_MoveDown"
+                        ToolTip="Move the selected mesh up. This uses an fbx export and import behind the scenes, make sure the settings in `File > Import > Replace Mesh...` are correct." >
+                    \/
+                </Button>
+                <Button DockPanel.Dock="Right" Margin="5" HorizontalAlignment="Right" Width="40" Height="25" Click="Mesh_MoveUp" 
+                        ToolTip="Move the selected mesh down. This uses an fbx export and import behind the scenes, make sure the settings in `File > Import > Replace Mesh...` are correct." >
+                    /\
+                </Button>
```

Comment:
Would be cooler if it was drag&drop instead. The switcher is offset based, so it certainly would be possible.